cclib/method/volume.py

```diff
@@ -17,21 +17,31 @@
 from cclib.parser.utils import find_package
 
 sym2powerlist = {

```

Comment:
Could you add a comment here explaining what this dictionary contains?

Reply:
I've added a short comment about this dict. Thanks.